app/src/main/java/com/myduka/app/activity/MainActivity.java

```diff
@@ -232,11 +233,11 @@ public void onResponse(okhttp3.Call call, okhttp3.Response response) throws IOEx
 
     public void getPhoneNumber() {
         AlertDialog.Builder builder = new AlertDialog.Builder(this);
-        builder.setTitle("Enter Customer's Safaricom phone number (2547XXX) to checkout Kshs " + String.valueOf(getTotal(prices)));
```

Comment:
I get the use of number sanitization, just add a comment to make sure developers know that all requests should be in the 254XXX format